editoast/editoast_derive/Cargo.toml

```
@@ -10,6 +10,8 @@ syn = "2.0"
 quote = "1.0"
 proc-macro2 = "1.0"
 darling = "0.20"
+serde_json = "1.0.108"
```

Use workspace to share the same version between crates.

```suggestion
serde_json.workspace = true
```

---

done